sdk/framework/backend.go

```diff
@@ -91,6 +91,9 @@ type Backend struct {
 	// BackendType is the logical.BackendType for the backend implementation
 	BackendType logical.BackendType
 
+	// RunningVersion is the optional version that will be self-reported
+	RunningVersion string
```

Comment:
Is this all that needs to be set in an external plugin once it's using the new sdk version? Like in the [newBackend() function](https://github.com/hashicorp/vault-plugin-secrets-kubernetes/blob/b423467b11688b76d1a873e806566f153ab51fbd/backend.go#L71), for instance?

Reply:
Yes. If a plugin sets this value, then that will be sufficient for the running version to appear in the mount/auth table.

They could also override the method directly, e.g., I tested this with vault-plugin-auth-azure:

```go
var _ logical.PluginVersioner = &azureAuthBackend{}

func (_ *azureAuthBackend) PluginVersion() logical.PluginVersion {
	return logical.PluginVersion{
		Version: "v1234",
	}
}
```